packages/arb-token-bridge-ui/src/components/TransferPanel/USDCWithdrawal/USDCWithdrawalConfirmationDialog.tsx

```diff
@@ -71,19 +70,16 @@ export function USDCWithdrawalConfirmationDialog(
             </button>
           </div>
 
-          <Tab.List className="bg-ocl-blue">
-            <TabButton>Use a fast bridge</TabButton>
-            <TabButton>Use Circle&apos;s bridge (USDC)</TabButton>
+          <Tab.List className="flex bg-ocl-blue">
+            <TabButton>Third party (USDC)</TabButton>
+            <TabButton>Circle (USDC)</TabButton>
           </Tab.List>
 
           <Tab.Panel className="flex flex-col space-y-3 px-8 py-4">
             <div className="flex flex-col space-y-3 font-light">
               <p>
-                Receive{' '}
-                <USDCTokenExplorerLink token="USDC" networkId={l1.network.id}>
-                  USDC
-                </USDCTokenExplorerLink>{' '}
-                on {toNetworkName} using a third-party bridge with Circle&apos;s{' '}
+                Receive <span className="font-bold">USDC</span> on{' '}
```

Comment:
we don't use 700 font weight (font-bold) on our UI
should be 500 `font-medium` instead

Reply:
Good point, updated